packages/core/lib/generators/assets_generator.dart

```diff
@@ -444,32 +446,34 @@ String _dotDelimiterStyleAssetsClassDefinition(
 ) {
   final statementsBlock =
       statements.map((statement) => statement.toStaticFieldString()).join('\n');
-  return _assetsClassDefinition(className, statements, statementsBlock);
+  final valuesBlock = _assetValuesDefinition(statements, static: true);
+  return _assetsClassDefinition(
+      className, statements, statementsBlock, valuesBlock);
 }
 
-String _assetValuesDefinition(List<_Statement> statements) {
+String _assetValuesDefinition(
+  List<_Statement> statements, {
+  /// Should the values definition be static
+  bool static = false,
```

Comment:
I think the argument is already self-explaining.

Reply:
As someone new to the project, I found a lot of the class names and function names unintuitive, so was generous at adding comments in this and other PRs.

Happy to remove this comment if you think it improves readability for a average contributor.

Reply:
Oh and also `static` seems to be a keyword. So maybe `staticDeclaring`?

Reply:
It is a [keyword but safe to use here](https://dart.dev/language/keywords).

But other choices are `staticDeclaration`, or shorter `staticField`?

Reply:
> But other choices are `staticDeclaration`, or shorter `staticField`?

I'm okay with both since I'm not a good name. :) Or we can just remove the comment.

Reply:
Ok removed comment. Please take a look.